Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -77,6 +77,17 @@
     sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/WinterCoats/coatcentcom.rsi
+    
+    - type: entity

```

Comment:
Please fix the indentation

Reply:
I think I fixed it, it's strange as there were no extra spaces when I first commited this